### PR TITLE
evmrs: add feature "dump-cov" and bash script to collect CT coverage data

### PR DESCRIPTION
**Pull request by LorenzSchueler**

### CODE REVIEW

**Comment by LorenzSchueler** [comment]
I'll wait until #826 is merged because I want to include documentation in BUILD.md for this but maybe @LuisPH3 you can have a look at in in the meantime anyway.

**Review by LuisPH3 — CHANGES_REQUESTED** [review]
When we implemented the C++ coverage, we agreed on adding a test to verify that the infrastructure is working.
The reason is that these mechanisms are rather flimsy and could silently stop working, specially when not used for a while. 

Please take a look on go/lib/cpp/coverage_test.go, and the last stage on the jenkins file. And try to repeat this patern. 
- build again with coverage enabled.
- run the go-test that checks that coverage data has been generated.

**Comment by LorenzSchueler** [comment]
The fix in the last commit is dirty but it was the only way I could find to get it to work.

**Review by LuisPH3 — APPROVED** [review]

**Pull request merged**
